stdlib/Cargo.toml

```diff
@@ -15,3 +15,7 @@ llvm-tools = "0.1.1"
 
 [lib]
 crate-type = ["staticlib", "rlib"]
+
+[features]
+default = []
+skip-range-support = []
```

Comment:
What do you think about these `disable-range-support`? That way for each set of RT features you use you can disable ones you don't support. I think the naming conveys intent better than `skip`.

Reply:
That makes sense, I can make that change.